src/components/Comments/Comment.js

```diff
@@ -306,7 +378,6 @@ const styles = StyleSheet.create({
     color: colorBasedVote
   }),
   btn: {
-    // width: 30,
     height: 30,
     justifyContent: 'center',
     alignItems: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The width property has been removed from the btn style. Make sure this doesn't affect the layout of the buttons in the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->